examples/gitlab/deploy-to-a-single-device.gitlab-ci.yml

```diff
@@ -33,20 +33,21 @@ mender:build:artifact:
   stage: build
   extends: .mender:build:artifact
   script:
-    - echo /home/root/.ssh > dest_dir
-    - echo authorized_keys > filename
-    - echo key > authorized_keys
+    # In this example, hello-mender.py represents a single file Python application that we have
+    # built and test in our CI and now we want to deploy into a device running Mender. We will
+    # package it as a Single File Application Update. For more information visit Mender docs:
+    # https://docs.mender.io/artifact-creation/create-an-artifact
+    - curl -O https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/examples/hello-mender.py
```

Comment:
will we need autoversion comment here to not fail ci/cd pipeline when we build docs site?

Reply:
No, that should be fine. `autoversion.py` will run in CI only for content in `mender-docs` repository.